Exercise2

schefs opened the pull request:
ready for review @eyalstoler

MargulisMax commented:
Eyal,

1. Remove commented lines:
# from pprint import pprint
# my_location = json.loads(requests.get('http://ip-api.com/json').content)
2. Initial docstring is missing
3. I don't think you that you need to output all the dict to a file but only the weather in a simple manner...
4. After running your code i noticed that the country value for Tel-Aviv in my_weather.txt is NI and not IL, check your get_weather function i noticed that you have no country code after city.
5. The code is easy and simple to read and understand .

schefs commented:
i added docstring and removed comented lines.
the openweatherapi has some faulty city-states references.NI to tel aviv is one of them,all other cities i tested work just fine, (you can query without a state as well, and see that tell aviv is not related to Israel also when querying weather with your browser GUI. )
